core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -90,7 +90,8 @@ describe('Topic editor functionality', function() {
     skillId = url.split('/')[4];
     await general.closeCurrentTabAndSwitchTo(handle);
     await topicsAndSkillsDashboardPage.get();
-    await topicsAndSkillsDashboardPage.navigateToUnusedSkillsTab();
+    await topicsAndSkillsDashboardPage.navigateToSkillsTab();
+    await topicsAndSkillsDashboardPage.filterSkillsByStatus('Unassigned');
```

Comment:
Could `Unassigned` be a global constant somewhere? Seems like this would come up a lot throughout the code, particularly the e2e tests.

Reply:
Can we decide how to do this later, since probably we'll have to create a constant.js file for the test files? (Pined regarding the same our hangouts group.)

Reply:
I agree with you suggestion to just make a constant in this file for now. Can you file an issue and put it in the QA team project so we don't forget?